src/main/java/frc/robot/commands/auto/DepositThenPath.java

```diff
@@ -12,21 +12,21 @@
 import frc.robot.constants.ElevatorConstants;
 import frc.robot.subsystems.Drivetrain;
 import frc.robot.subsystems.Elevator;
-import frc.robot.subsystems.FourBarArm;
 import frc.robot.subsystems.RollerIntake;
+import frc.robot.subsystems.Wrist;
 import frc.robot.util.GamePieceType;
 
 public class DepositThenPath extends SequentialCommandGroup {
-  public DepositThenPath(String pathName, Position depositPosition, Drivetrain drive, Elevator elevator, FourBarArm arm, RollerIntake intake) {
-    addRequirements(drive, elevator, arm, intake);
+  public DepositThenPath(String pathName, Position depositPosition, Drivetrain drive, Elevator elevator, Wrist wrist, RollerIntake intake) {
+    addRequirements(drive, elevator, wrist, intake);
     addCommands(
       new CalibrateElevator(elevator),
-      depositPosition == Position.MIDDLE ?
-        new PositionIntake(elevator, arm, () -> false, depositPosition).withTimeout(1.5) : 
-        new MoveElevator(elevator, ElevatorConstants.kMiddleConeHeight).withTimeout(1).andThen(new PositionIntake(elevator, arm, () -> false, Position.TOP).withTimeout(1.5)),
+      depositPosition == Position.TOP ?
+        new MoveElevator(elevator, ElevatorConstants.kMiddleConeHeight).withTimeout(1).andThen(new PositionIntake(elevator, wrist, () -> false, Position.TOP).withTimeout(1.5)) :
+        new PositionIntake(elevator, wrist, () -> false, depositPosition).withTimeout(1.5),
```

Comment:
if the elevator is at top it moves elevator to middle and then to top, is there a reason for this.

Reply:
sequencing for top node middle node is to rduc etime and wrist cna start rotateing.

Reply:
could use a do nothing command instead of havieng an andThen decorator and move the position intake outside